src/main/app/Resources/translations/functional.en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "addBadge": "The badge badgeName has been added to the user userName",
+  "removeBadge": "The badge badgeName has been removed to the user userName",
+  "resourceOpen": "The resource resourceName has been opened by user userName",
+  "resourceEvaluation": "resourceName - statusName : userName is currently in step userProgression for a duration of durationTime sec",
+  "resourceScore": "resourceName - statusName : userName scored currentScore out on scoreMax. The min score is scoreMin",
+  "toolOpen": "context : User userName open the tool toolName"
+}
```

Comment:
This should be moved in correct domain (badge, resource, tool, etc.) and removed.